### ISSUE TITLE
Replace the rule eslint/no-shadow by @typescript-eslint/no-shadow if typescript-eslint plugin is extended

### ISSUE THREAD

**Issue by JordanLao** [theodo/clinter#13]
The `@typescript-eslint/no-shadow` rule extends the `eslint/no-shadow`. Having both activated can cause incorrect errors :

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md

To fix this error, if `typescript-eslint` plugin is extended, the following lines must be added to the eslint configuration file :

```
{
  // note you must disable the base rule as it can report incorrect errors
  "no-shadow": "off",
  "@typescript-eslint/no-shadow": ["error"]
}
```

**Comment by CorentinDoue** [comment]
fixed with https://github.com/theodo/clinter/pull/27
